templates/config.tfvars.openstack-template

```diff
@@ -5,6 +5,7 @@ kubeadm_token = "your-kubeadm-token" # Autogenerated kubeadm token (don't change
 floating_ip_pool = "your-pool-name"
 external_network_uuid = "external-net-uuid" # The uuid of the external network in the OpenStack tenancy
 boot_image = "kubenow-v050"
+disc_format = "raw"
```

Comment:
this should be commented, or defaulting to `qcow`

Reply:
yes, I will default it to "qcow2" .. changing.